src/sonic-bgpcfgd/tests/test_bgp.py

```diff
@@ -4,15 +4,23 @@
 from bgpcfgd.directory import Directory
 from bgpcfgd.template import TemplateFabric
 from . import swsscommon_test
-from .util import load_constants
+from .util import load_constants, CONSTANTS_PATH
 from swsscommon import swsscommon
 import bgpcfgd.managers_bgp
 
 TEMPLATE_PATH = os.path.abspath('../../dockers/docker-fpm-frr/frr')
 
-def constructor():
+def load_constant_files():
+    path = "tests/data/constants"
+    constant_files = [os.path.abspath(os.path.join(path, name)) for name in os.listdir(path)
```

Comment:
@geogchen 
For the constants file which is included in the image, I think we should also include it in the testing for the verification, otherwise, the illegal change in the files/image_config/constants/constants.yml won't be catched up by the UT.

Reply:
Changed to include constants.yml under image_config/constants